database_cleanup/tests/common.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.modules.registry import Registry
+from odoo.tests import TransactionCase
+from odoo.tests.common import tagged
+
+
+# Use post_install to get all models loaded more info: odoo/odoo#13458
+@tagged("post_install", "-at_install")
```

Comment:
I think something may be wrong here.

Note that the [normal tests fail](https://app.travis-ci.com/github/OCA/server-tools/jobs/557882113#L3323) before the [post tests get started](https://app.travis-ci.com/github/OCA/server-tools/jobs/557882113#L4047)

But the `database_cleanup` test should be part of the post installation tests, because otherwise, not all modules are loaded (eg. `base_custom_info` is loaded [as number 53](https://app.travis-ci.com/github/OCA/server-tools/jobs/557882113#L3619)) before `database_cleanup` is loaded (as [number 22, so earlier](https://app.travis-ci.com/github/OCA/server-tools/jobs/557882113#L1930)) and `database_cleanup` will try to delete tables belonging to not-yet-loaded modules instead of deprecated modules and you get these errors.

So, I suspect that with Odoo 14.0, `@tagged` perhaps does not propagate anymore to child tests and we should define it on all tests specifically.

Reply:
@StephaneMangin